knowlg-automation/terraform/aws/vpc.tf

```diff
@@ -0,0 +1,126 @@
+resource "aws_vpc" "vpc" {
+  cidr_block           = var.vpc_cidr
+  instance_tenancy     = "default"
+  enable_dns_hostnames = true
+  tags = merge(
+    local.common_tags,
+    var.additional_tags
+  )
+}
+
+resource "aws_internet_gateway" "gateway" {
+  vpc_id = aws_vpc.vpc.id
+  tags = merge(
+    local.common_tags,
+  var.additional_tags)
+}
+
+resource "aws_eip" "nat" {
```

Comment:
Why do we need a eip here?

Reply:
we also created Nat gateway so it is just to allocate public ip to nat as then only we can use the them in the Nat